Jenkinsfile

```diff
@@ -228,16 +227,11 @@ def validateReleaseVersion() {
 
 def downloadToolsCert() {
     stage('Download tools cert') {
-        // Download the certificate file and key file, used for signing the JFrog CLI binary.
-        withCredentials([
-            string(credentialsId: 'download-signing-cert-access-token', variable: 'DOWNLOAD_SIGNING_CERT_ACCESS_TOKEN'),
-            string(credentialsId: 'repo21-url', variable: 'REPO21_URL')
-        ]) {
+        // Download the certificate files, used for signing the JFrog CLI binary.
+        // To update the certificate, download digicert_sign.zip and follow the instructions in the README file.
```

Comment:
To update the certificate, download digicert_sign.zip and follow the instructions in the README file.
-->
To update the certificate before it is expired, download the digicert_sign.zip file and follow the instructions in the README file, which is packaged inside that zip.